doc/library-contributor.rst

```diff
@@ -16,10 +16,10 @@ Release Notes
 Major and Minor Releases
 ........................
 
-When making a commit to a Neptune-versioned library, API-affected changes should
-be noted in a file in the ``relnotes`` folder, as follows. When the
-corresponding branch is released, the files in that folder will be collated to
-form the notes for the release.
+When making a commit to a Neptune-versioned library, changes that are not purely
+internal implementation details should be noted in a file in the ``relnotes``
+folder, as follows. When the corresponding branch is released, the files in that
+folder will be collated to form the notes for the release.
```

Comment:
I found the mentioning of API very clear compared to this

Reply:
Yeah, I'll remove this change. I think "API-affecting" is totally clear. (There was some discussing around whether "extending" is covered by "affecting". I believe it is.)